### PR TITLE
chore: added emojis, fixed typos and grammatical errors and spaced text out

### PR DESCRIPTION
**Pull request by HareerSakhi**
Please check my changes. Thanks (:

### CODE REVIEW

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @HareerSakhi,
Please do not change the text to the title, only add the emoji or color

**Comment by HareerSakhi** [comment]
I have removed all of the text size changes. Thanks

File: english/normal-dungeon/4/4.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-You open the door to your neighbour's house, and the first thing you see is a trail of blood💉 streaked on the floor...
+ You open the door 🚪 to your neighbour's house 🏡, and the first thing you see is a <span style= "color:red">trail of blood💉 streaked on the floor...</span>
```
Comment (ming-tsai) [inline english/normal-dungeon/4/4.md:1]: DO NOT use HTML code on the markdown file. if you are more advanced and know ReactJs you could beautify the page

**Review by ming-tsai — CHANGES_REQUESTED** [review]

**Comment by HareerSakhi** [comment]
I have gotten rid of the inline HTML, and made the page more visible by spacing out the text a little bit, added more emojis, and fixed some typos and grammar errors that I spotted. Thanks

**Comment by HareerSakhi** [comment]
I have gotten rid of the inline HTML, and made the page more visible by spacing out the text a little bit to make the writing more visible, added more emojis, and fixed some typos and grammar errors that I spotted. Thanks.

**Comment by HareerSakhi** [comment]
I have also completed folder 5 of the english normal-dungeon, adding more emojis (and also adding spaces before them), fixing grammatical errors, fixed one or two of the sentences so that they make more sense, and spacing text out.

**Review by ming-tsai — APPROVED** [review]
Hi @HareerSakhi,
Nice work, and welcome to be part of the team.
> The invitation [link](https://github.com/MakeContributions)

**Pull request merged**